libmscore/read301.cpp

```diff
@@ -166,7 +166,7 @@ bool Score::read(XmlReader& e)
                   else {
                         e.tracks().clear();     // ???
                         MasterScore* m = masterScore();
-                        Score* s       = new Score(m, false);
+                        Score* s       = new Score(m, MScore::baseStyle());
```

Comment:
As discussed on IRC, we believe this is the right thing to do.  But I a concerned that constructor being referenced here (the one taking a MasterScore and an MStyle) is actually *removing* the score from validScores rather than adding to it.  @dmitrio95 , was this an oversight?  This seems more appropriate for the destructor than for a constructor.

Reply:
Yes, I am really surprised that this removal is in a constructor... but apparently I put it there. Sure, it should rather go to the destructor, constructors should add a score to the valid scores list. @peterhieuvu feel free to correct that in this pull request if it is needed for your change to work properly. Otherwise I can create a separate PR to correct that issue.

Reply:
Sounds good, thank you! I added to this PR, changing the constructor to add to validScores and erase from validScores in the destructor.

Reply:
Great, thank you!

Reply:
Yes, this should be good.  For the record, the existing validScores code didn't cause the fix here to not work, it just meant we lose the advantage of what it was doing to help prevent crashes on element destruction (because any score read as opposed to created from scratch would not be in the validScores list).

So to me this is ready to merge, and should be top priority, definitely in master, but also should we elect to do a 3.0.6 (and indeed could be one strong reason for doing one, if 3.1 is going to be more than a few weeks away).